docs/developers/NewToGluten.md

```diff
@@ -102,18 +103,21 @@ If you have Ultimate intellij, you can try to debug remotely.
 - Activate your profiles such as <backends-velox>, and Reload Maven Project, you will find all your need modules have been activated.
 - Create breakpoint and debug as you wish, maybe you can try `CTRL+N` to find `TestOperator` to start your test.
 
-# Debug cpp code with Visual Studio Code
+## Java/Scala code style setting
+
+Intellij IDE supports importing settings for Java/Scala code style. You can import [intellij-codestyle.xml](../../dev/intellij-codestyle.xml) to your IDE.
+See [Intellij guide](https://www.jetbrains.com/help/idea/configuring-code-style.html#import-code-style).
```

Comment:
That's great!

Could you use this style do a whole project re-format? It would bring much benefits for following PR.

Reply:
Yes, I will do that.